docker/compose/docker-compose.mariadb.yml

```diff
@@ -54,11 +54,6 @@ services:
       - broker
     ports:
       - "8000:8000"
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8000"]
```

Comment:
Incidentally the mariadb health check were slightly different, but as far as I can tell this was not intentional